.github/workflows/macos.yaml

```diff
@@ -129,8 +129,8 @@ jobs:
       env:
         CODESIGNING_CERT_BASE64: ${{ secrets.CODESIGNING_CERT_B64 }}
         CODESIGNING_CERT_PASSPHRASE: ${{ secrets.CODESIGNING_CERT_PASSPHRASE }}
-        INSTALLER_CERT_BASE64: ${{ secrets.INSTALLER_CERT_B64 }}
-        INSTALLER_CERT_PASSPHRASE: ${{ secrets.INSTALLER_CERT_PASSPHRASE }}
+        INSTALLER_CERT_BASE64: ${{ secrets.INSTALLER_CERT_2024_B64 }}
+        INSTALLER_CERT_PASSPHRASE: ${{ secrets.INSTALLER_CERT_2024_PASSPHRASE }}
```

Comment:
Can't we substitute the secret itself instead of renaming and adding it?

Reply:
I think renaming it is a better practice because it allows to control the change with Pull Requests. It also allows versioning and revert in case anything breaks.